src/client/gui/lib/platform/linux.dart

```diff
@@ -32,6 +33,12 @@ class LinuxPlatform extends MpPlatform {
             CopySelectionTextIntent.copy,
         SingleActivator(LogicalKeyboardKey.keyV, control: true, shift: true):
             PasteTextIntent(SelectionChangedCause.keyboard),
+        SingleActivator(LogicalKeyboardKey.equal, control: true):
+            IncreaseTerminalFontIntent(),
```

Comment:
These 4 intents exist for each platform. Maybe it would be better to define default terminal shortcuts in mpPlatform?

```
  // Define default terminal shortcuts (platform.dart)
  Map<SingleActivator, Intent> get terminalShortcuts => {
        SingleActivator(LogicalKeyboardKey.keyC, control: true): CopySelectionTextIntent.copy,
        SingleActivator(LogicalKeyboardKey.keyV, control: true, shift: true): PasteTextIntent(SelectionChangedCause.keyboard),
        SingleActivator(LogicalKeyboardKey.equal, control: true): IncreaseTerminalFontIntent(),
        SingleActivator(LogicalKeyboardKey.minus, control: true): DecreaseTerminalFontIntent(),
        SingleActivator(LogicalKeyboardKey.digit0, control: true): ResetTerminalFontIntent(),
      };
```

Reply:
Well, they are not really the same, because macos uses Command (`meta` in this case) instead of Control for those shortcuts. And the fact that linux and windows share the shortcuts is more of a coincidence than a duplication. Since changing one does not imply that the other has to change as well, it is not duplication IMO.